zookeeper-recipes/zookeeper-recipes-lock/src/main/java/org/apache/zookeeper/recipes/lock/ZNodeName.java

```diff
@@ -102,9 +125,9 @@ public String getName() {
     }
 
     /**
-     * Returns the sequence number.
+     * Returns the optional sequence number.
      */
-    public int getZNodeName() {
+    public Optional<Integer> getSequence() {
```

Comment:
it looks like this method is called only in tests.
you can drop the 'public' modifier

Reply:
All of the other `getters` are also public, so I think it should be consistent with them.